confusing error message if `OPENDP_LIB_DIR` is set to ""

Not sure how or why I set it to an empty string, but it is:
```
>>> os.environ.get("OPENDP_LIB_DIR")
''
```

The empty string becomes `.` when wrapped in a path:
```
>>> Path('')
PosixPath('.')
```

This causes a confusing error message from pytest:
```
    raise Exception(f"Expected exactly one binary to be present in {lib_dir}. Got: {lib_dir_file_names}")  # pragma: no cover
E   Exception: Expected exactly one binary to be present in .. Got: []
```

`unset OPENDP_LIB_DIR` fixes it for me, but we could also make this a little more robust.

--- a/python/src/opendp/_lib.py
+++ b/python/src/opendp/_lib.py
@@ -20,7 +20,8 @@ ATOM_EQUIVALENCE_CLASSES: MutableMapping[str, Sequence[str]] = {
 
 def _load_library():
     default_lib_dir = Path(__file__).absolute().parent / "lib"
-    lib_dir = Path(os.environ.get("OPENDP_LIB_DIR", default_lib_dir))
+    lib_envvar = "OPENDP_LIB_DIR"
+    lib_dir = Path(os.environ.get(lib_envvar, default_lib_dir))
     if not lib_dir.exists():
         # fall back to default location of binaries in a developer install
         build_dir = 'debug' if os.environ.get('OPENDP_TEST_RELEASE', "false") == "false" else 'release'
@@ -36,8 +37,10 @@ def _load_library():
             and any(p.name.endswith(suffix) for suffix in suffixes)
         ]
         
-        if len(lib_dir_file_names) != 1:
-            raise Exception(f"Expected exactly one binary to be present in {lib_dir}. Got: {lib_dir_file_names}")  # pragma: no cover
+        if len(lib_dir_file_names) != 1:  # pragma: no cover
+            value = os.environ.get(lib_envvar)
+            envvar_info = f" ({lib_envvar}='{value}')" if value is not None else ""
+            raise Exception(f"Expected exactly one binary to be present in '{lib_dir}'{envvar_info}. Got: {lib_dir_file_names}")
         
         lib_path = lib_dir / lib_dir_file_names[0]
         try:

